webapi/src/main/scala/org/knora/webapi/responders/v2/OntologyResponderV2.scala

```diff
@@ -37,6 +38,11 @@ import org.knora.webapi.responders.v2.ontology.Cache
 import org.knora.webapi.responders.v2.ontology.Cache.ONTOLOGY_CACHE_LOCK_IRI
 import org.knora.webapi.responders.v2.ontology.CardinalityHandler
 import org.knora.webapi.responders.v2.ontology.OntologyHelpers
+import org.knora.webapi.responders.v2.ontology.OntologyHelpers.isFileValueProp
+import org.knora.webapi.responders.v2.ontology.OntologyHelpers.isKnoraResourceProperty
+import org.knora.webapi.responders.v2.ontology.OntologyHelpers.isLinkProp
+import org.knora.webapi.responders.v2.ontology.OntologyHelpers.isLinkValueProp
```

Comment:
Imports are a matter of personal preference. I do not have a strict rule when or how functions  or the object containing them should be imported. Mostly I am improving for readability at the place I am using these functions. 

The whole `OntologyHelpers` file/object is a bit odd anyway. Its naming does not reveal the purpose, so adding the object at the places where it is used does not add much of valuable information to the code there, it does not hurt either if the space is there.

IMHO this is more of a personal style decision than anything that should prevent PR to be merged, wdyt?

Reply:
It's true that is personal perception, but as we are working here together we should align on something, otherwise in every file touched by someone with different perception one would find different code structure which IMO makes code hard to read. We can bring this topic into BE meeting discussion.

Reply:
You have my full support on us as a team on aligning on coding conventions.

I'd be happy to talk about that in a future meeting but  would not want to wait for this PR not to be merged until we have made a decision. Currently there are not convention and this should not stop us from integrating changes.

Reply:
Deal.